### DISCUSSION TITLE
Windows Defender goes nuts with downloaded 362 zip

### DISCUSSION THREAD

**Discussion by jkr4m3r** [ezQuake/ezquake-source#754] [General]
Windows Defender goes nuts with downloaded 362 zip upon download.

VT comes up clean. 

Defender claims a trojan called **Wacatac. B! ml**

**Comment by tcsabina** [comment]
Hi @jkr4m3r ,

Yeah, I've heard so far from 2 users in the QW Discord that indeed Defender thinks it has a trojan. But many other users reported that this is a false positive.
This is the first time I zipped the binary. In previous releases we just added the .exe directly. Probably I will do it here as well.

**Comment by jkr4m3r** [comment]
Similar here, the zip was freaking it out, and after I fought it and got it unzipped it was fine with it on both VT and locally with defender. Weird.

**Comment by tcsabina** [comment]
alright. no more zipping ;).

I've updated the Release: removed the .zip and added the .exe.
Will do it with upcoming releases as well.

**Comment by jkr4m3r** [comment]
I was never concerned about it being malicious, I'm concerned with Windows defender being a pile of sh!t.
